### ISSUE TITLE
Remove portality

### ISSUE THREAD

**Issue by jpwanabe** [ChosenArchitect/Project-Architect#162]
**Is your feature request related to a problem? Please describe.**
->The mod is causing a lot of crashes

**Describe the solution you'd like**
->Remove it from the pack

**Describe alternatives you've considered**
->Block the crafting and usage of the blocks

**Mod Sugestions**
->

**Additional context**
->One of many crash logs involving it.
https://pastebin.com/zwQXeKWa

**Comment by James103** [comment]
1. How did you reproduce the crash?
2. Can you please post all of your crash logs that have to do with the Portality mod?

**Comment by jpwanabe** [comment]
Wish I could tell you. It crashes any time the chunk is loaded. I had to remove the mod from the server so the could start and be playable. All crash logs show the same error as the one I posted. Just dif cords on line 36 and 37.

**Comment by James103** [comment]
Can you please reproduce this on a fresh new world? If it crashes there, can you please attach your world file?

**Comment by jpwanabe** [comment]
I do not know what the player did to cause it, and neither do they. They said they had the portal extra long and I have linked a pick of what I got before the server crashed the last time. And this is after trying to remove it in unloaded chunks with commands. 
https://i.imgur.com/vclhbab.png


If I had to guess, either the amount of flux points, the one inside the frame, or the arrows that were everywhere in side the frame was the cause.
